src/gui_tool/entry.py

```diff
@@ -50,11 +50,26 @@ def split_file(file_loc, mdi:MetaDataItem):
             context = GUIContext(file_loc)
             gui = SPGUIManager(context)
             rs = gui.start()
-            ret = []
-            for start, end in rs:
-                m = mdi.clone()
+            ret = [mdi]
+
+            split_vid = False
+            if len(rs) > 1:
+                split_vid = True
+
+            for i, vid_range in enumerate(rs):
+                start, end = vid_range
+
+                if i == 0:
+                    m = mdi
```

Comment:
@jackwilliamson 
If I understanding the logic here, if len(rs) == 0, we get ret = [mdi, mdi]

Reply:
len(rs) can be 0, so nothing should get passed on.

I think it'd be best to have the same logic as before, except 1) mdi.is_cancelled is set to True, and 2) ret =[mdi] at the beginning.

Reply:
If rs is empty, does that mean we want the entire video, or that we don't want to use this video, because these changes wouldn't send anything other than a canceled_video in that circumstance.

Reply:
And while we can also just cancel the original mdi, I feel like we can avoid leaving a bunch of canceled metadata items in the database if we just ensure that the first range is given to mdi.

Reply:
rs is empty implies the user decided no section would be passed on. rs only contains the sections that the user wants to keep

So a "first range" might not exist

Reply:
I see, so in the case that rs is empty, then the video should be canceled.

Reply:
Yeah, that works too. We just do a len check if you want